tests/bsim/bluetooth/host/l2cap/split/tester/prj.conf

```diff
@@ -3,7 +3,7 @@ CONFIG_ASSERT=y
 
 CONFIG_BT=y
 CONFIG_BT_HCI_RAW=y
-CONFIG_BT_MAX_CONN=16
+CONFIG_BT_MAX_CONN=1
```

Comment:
Why do we have so many tests and samples that use `CONFIG_BT_MAX_CONN=16`

Reply:
idk ask the OPs. The answer is probably lots of blind copy-pasting.